yaraneba/spec/requests/task_request_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe 'Tasks', type: :request do
+  before do
+    @task = FactoryBot.create(:task)
+  end
+
+  describe '#task index' do
+    context 'GET' do
+      example 'request OK' do
+        get root_path
+        expect(response.status).to eq(200)
+      end
+    end
+  end
+
+  describe '#task new' do
+    context 'GET' do
+      example 'request OK' do
+        get new_task_path
+        expect(response.status).to eq(200)
+      end
+    end
+    context 'POST' do
+      example 'request OK' do
+        post tasks_path, params: { task: FactoryBot.attributes_for(:task) }
```

Comment:
FactoryBot っていう箇所はいらなそう
```suggestion
        post tasks_path, params: { task: attributes_for(:task) }
```